buildspec-unittest-python3.6.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      # install python 3.6 because it is not available on codebuild anymore
```

Comment:
same here, but really depends on how long the installation takes.

Reply:
It only takes 4 mins to install and run the test